bindings/java/README.md

```diff
@@ -6,6 +6,52 @@
 
 ![](https://github.com/apache/incubator-opendal/assets/5351546/87bbf6e5-f19e-449a-b368-3e283016c887)
 
+## Example
+```java
+import org.apache.opendal.Metadata;
+import org.apache.opendal.Operator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.apache.opendal.test.behavior.BehaviorTestBase.generateBytes;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
Of course a user-facing hello world should not use test facilities.

Reply:
Thank you for the suggestion and review, I will revise this